src/includes/home/banner.njk

```diff
@@ -4,71 +4,39 @@
-                />
-              </svg>
-            </span>
+            <a href="/careers/">
+              <span class="flex p-2 rounded-lg bg-blue-800">
+                <!-- Heroicon name: heart -->
+                <svg class="h-6 w-6 text-white" xmlns="http://www.w3.org/2000/svg" fill="none" viewbox="0 0 24 24" stroke="currentColor">
+                  <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4.318 6.318a4.5 4.5 0 000 6.364L12 20.364l7.682-7.682a4.5 4.5 0 00-6.364-6.364L12 7.636l-1.318-1.318a4.5 4.5 0 00-6.364 0z"/>
+                </svg>
+              </span>
             </a>
             <p class="ml-3 font-medium text-white truncate">
               <span class="md:hidden">
-                <a href="/blog/powering-community-led-growth/">
-                We’ve raised a $4M seed round
+                <a href="/careers/">
+                  Come grow with us! We're looking for awesome people to join the Orbit team!
                 </a>
               </span>
               <span class="hidden md:inline">
-                <a href="/blog/powering-community-led-growth/">
-                  We've raised a $4M seed round to power the future of community-led growth
+                <a href="/careers/">
+                  Come grow with us! We're looking for awesome people to join the Orbit team!
                 </a>
               </span>
             </p>
           </div>
-          <div
-            class="order-3 mt-2 flex-shrink-0 w-full sm:order-2 sm:mt-0 sm:w-auto"
-          >
+          <div class="order-3 mt-2 flex-shrink-0 w-full sm:order-2 sm:mt-0 sm:w-auto">
             <div class="rounded-md shadow-sm">
-              <a
-                href="/blog/powering-community-led-growth/"
-                class="flex items-center justify-center px-4 py-2 border border-transparent text-sm leading-5 font-medium rounded-md text-blue-600 bg-white hover:text-blue-500 focus:outline-none focus:shadow-outline transition ease-in-out duration-150"
-              >
-                Learn more
+              <a href="/careers/" class="flex items-center justify-center px-4 py-2 border border-transparent text-sm leading-5 font-medium rounded-md text-blue-600 bg-white hover:text-blue-500 focus:outline-none focus:shadow-outline transition ease-in-out duration-150">
+                See open positions
               </a>
             </div>
           </div>
           <div class="order-2 flex-shrink-0 sm:order-3 sm:ml-2">
-            <button
-              type="button"
-              class="-mr-1 flex p-2 rounded-md hover:bg-blue-500 focus:outline-none focus:bg-blue-500 transition ease-in-out duration-150"
-              aria-label="Dismiss"
-              @click="visible = false"
-            >
+            <button type="button" class="-mr-1 flex p-2 rounded-md hover:bg-blue-500 focus:outline-none focus:bg-blue-500 transition ease-in-out duration-150" aria-label="Dismiss" @click="visible = false">
```

Comment:
@alexlsalt Just checking if these changes are updating functionality or maybe made it into the commit accidentally, perhaps due to some auto-formatting?

Reply:
Oh my goodness - sorry about that! Yeah, it ended up being formatted on save, so I just fixed everything so that only the necessary changes were added 👍